.clean

```diff
@@ -0,0 +1,21 @@
+Build
+build
+Debug
+Release
+lib
+__pycache__
+.idea
+.*.sw[po]
+*~
+*.pyc
+*.pyd
+*.pyo
+*.egg-info
+dist
+setuptools*egg
+setuptools.pth
+distribute*egg
+distribute*tar.gz
+*.so
+*.o
+*.log
```

Comment:
Could this list be built into the code.  I'm not sure I like the idea of an external file list.

Reply:
Yes I could but it'll be ugly, one other choice is to load .gitignore than .clean